packages/react-color-picker/src/lib/webgl.ts

```diff
@@ -243,7 +243,7 @@ void main() {
   // 3 = projection toward point, hue dependent
   // 4 = adaptive Lightness, hue independent
   // 5 = adaptive Lightness, hue dependent
-  int clamp_mode = 3;
+  int clamp_mode = 2;
```

Comment:
Trying to find a good balance between weird Lightness & weird Chroma scales. 

Actual fix may be more involved, but just experimenting for now